mappings/net/minecraft/network/PacketByteBuf.mapping

```diff
@@ -601,6 +609,12 @@ CLASS net/minecraft/unmapped/C_idfydwco net/minecraft/network/PacketByteBuf
 		COMMENT @return this buf, for chaining
 		ARG 1 array
 			COMMENT the array to write
+	METHOD m_ngsogwlc decodeById (Ljava/util/function/IntFunction;)Ljava/lang/Object;
+		COMMENT Reads a {@linkplain #readVarInt var int} from this buf,
+		COMMENT and then runs {@code idToValue} to convert it to a value.
```

Comment:
seems strange to talk about implementation here, maybe tag with `@implNote` if the implementation really needs to be discussed

Reply:
I think it needs the information, since you're passing in a function